app/controllers/articles_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
 
   def create
     @article = Article.new(article_params)
-    @article.author = current_user.username
+    @article.user = current_user
```

Comment:
In the case of a `has_many` relationship, you get access to scoped builder & finder methods:
```ruby
# Builder
@article = current_user.articles.build(article_params)

# Finder
current_user.articles.find(params[:id])
```